src/helpdesk_proj/static/helpdesk_app/css/search.css

```diff
@@ -205,4 +205,25 @@ li a:hover {
 
 .result-highlighted {
     background-color: #1f1e1e;
+}
+
+.faq-box {
+    width: fit-content;
```

Comment:
As part of #78 efforts, can we change this to `width: 80%;` so that it looks cleaner on mobile?

Reply:
For me it looks reasonable on mobile. With `width: 80%` I think it looks a little weird on desktop.

Current mobile:
![image](https://user-images.githubusercontent.com/11011325/234358028-ca5cc01f-0514-4033-93d3-f5bd4a3ab6f0.png)

Updated desktop:
![image](https://user-images.githubusercontent.com/11011325/234358171-67d0a90c-4152-4d30-b5a7-7ee70a9ca661.png)

I can revisit this after this PR and #78 is merged.